internal/validator/rules/alpha.go

```diff
@@ -37,9 +37,18 @@ func (v *alphaValidator) Err() string {
 
 	validator.GeneratorMemory[fmt.Sprintf(alphaKey, v.FieldName())] = true
 
-	return strings.ReplaceAll(`
-    // Err@AlphaValidation is the error returned when field @ is not alphabetic.
-    Err@AlphaValidation = errors.New("field @ must be alphabetic")`, "@", v.structName+v.FieldName())
+	const errTemplate = `
+		// [ERRVARIABLE] is the error returned when field @ is not alphabetic.
+		[ERRVARIABLE] = govaliderrors.ValidationError{Reason:"field @ must be alphabetic",Path:"PATH"}
+	`
+
+	replacer := strings.NewReplacer(
+		"[ERRVARIABLE]", v.ErrVariable(),
+		"@", v.FieldName(),
+		"PATH", fmt.Sprintf("%s.%s", v.structName, v.structName+v.FieldName()),
+	)
```

Comment:
I feel like there are 3 pattern here. Only one should be used

```suggestion
	replacer := strings.NewReplacer(
		"[ERRVARIABLE]", v.ErrVariable(),
		"[FIELD]", v.FieldName(),
		"[PATH]", fmt.Sprintf("%s.%s", v.structName, v.structName+v.FieldName()),
	)
```

I report it because I had always found @ strange and now others are added, it feel like it should be fixed and consistent.

Also, I would prefer avoiding a pattern that could collide with field name, so I would prefer something that couldn't break later if description/comments are added and autogenerated. I'm afraid something like "this variable would be use in PATH" with the logic of Unix PATH variable, and it would be replaced

This format is more likely to never collide with something the user may provide

```suggestion
	replacer := strings.NewReplacer(
		"[@ERRVARIABLE]", v.ErrVariable(),
		"[@FIELD]", v.FieldName(),
		"[@PATH]", fmt.Sprintf("%s.%s", v.structName, v.structName+v.FieldName()),
	)
```

Reply:
> I report it because I had always found @ strange and now others are added, it feel like it should be fixed and consistent.
Agree better maintain consistency with the @ that was already in there.
If after this one it doesn't look good it can be changed later on.

Reply:
[76c06d9](https://github.com/sivchari/govalid/pull/121/commits/76c06d91c0787734ab214f514301da2f5edc5792) solved with